.gitignore

```
@@ -1,6 +1,8 @@
-__pycache__
-public/*.html
-public/*.svg
-
+GITHUB_TOKEN
+ZENHUB_TOKEN
+__pycache__/
 github_schema.json
 github_schema.py
+poetry.lock
+public/zcash-tfl-dag.html
+public/zcash-tfl-dag.svg
```

```suggestion
public/
```
Everything under `public/` is generated. (These are also not the only relevant filenames; it's actually `zcash-*-dag.{html,svg}`.)